code/datums/wires/syndicatebomb.dm

```diff
@@ -106,6 +107,13 @@
 				if(isliving(usr))
 					add_memory_in_range(B, 7, /datum/memory/bomb_defuse_success, protagonist = usr, antagonist = B, bomb_time_left = bomb_time_left)
 
+/datum/wires/syndicatebomb/can_reveal_wires(mob/user)
+	. = ..()
+	if(.)
+		return .
+	return is_syndicate_affiliated(user)
```

Comment:
```suggestion
	return ..() is_syndicate_affiliated(user)
```

Reply:
After considering your review I've realized what I believe is a better way to do this entirely versus fixing up what I have now